sway-lib-std/src/hash.sw

```diff
@@ -124,3 +124,31 @@ pub fn hash_pair(value_a: b256, value_b: b256, method: HashMethod) -> b256 {
         }
     }
 }
+
+// Registers are 64 bits (8 bytes) wide. Words are the same width as registers.
```

Comment:
Use triple backslash `///`for public comments as they will eventually be used for auto-generated docs.
Give some description of what this function is for.